### PR TITLE
Adding standalone section about applying changes and linking it from …

### PR DESCRIPTION
**Pull request by jhutar**
…various places of the tuning guide

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_applying-configurations.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+[id="Applying configurations_{context}"]
+= Applying configurations
+
+In following sections we suggest various tunables and how to apply them.
+Please always test changing these in non production environment first, with valid backup and with proper outage window as in most of the cases Satellite restart is required.
+
+It is also a good practice to setup a monitoring before applying any change as it will allow you to evaluate effect of the change.
+Our testing environment might be too far from what you will see although we are trying hard to mimic real world environment.
+
+If you have changed some SystemD service file, you need to notify SystemD daemon to reload the configuration:
```
Comment (ekohl) [inline guides/common/modules/proc_applying-configurations.adoc:10]: Systemd is all lowercase.
```suggestion
If you have changed some systemd service file, you need to notify systemd daemon to reload the configuration:
```

**Review by ekohl — COMMENTED** [review]
Not a full review, just a small note.

File: guides/common/modules/con_performance-tuning-quickstart.adoc [hunk 2]
```diff
@@ -26,5 +26,6 @@ The default value is `default`.
 ----
 # {foreman-installer} --tuning "_My_Installation_Size_"
 ----
+. Optional: There are some additional checks you can do in <<Applying configurations_{context}>> section.
```
Comment (maximiliankolb) [inline guides/common/modules/con_performance-tuning-quickstart.adoc:29]: Please use `xref:_My_Anchor_{context}[_My_Title_]`.

We generally say "For more information, see xref:...."

File: guides/common/modules/proc_applying-configurations.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Check number of fact names in database:                               [OK]
+--------------------------------------------------------------------------------
+Check whether all services are running:                               [OK]
+--------------------------------------------------------------------------------
+Check whether all services are running using the ping call:           [OK]
+--------------------------------------------------------------------------------
+Check for paused tasks:                                               [OK]
+--------------------------------------------------------------------------------
+Check whether system is self-registered or not:                       [OK]
+--------------------------------------------------------------------------------
```
Comment (maximiliankolb) [inline guides/common/modules/proc_applying-configurations.adoc:53]:
```suggestion
```

We generally don't include console output. Instead, you can say "Verify that all services are running:
command ...
"

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/con_performance-tuning-quickstart.adoc (hunk 2 above)
Comment (jhutar) [reply guides/common/modules/con_performance-tuning-quickstart.adoc]: Should be fixed, thank you

File: guides/common/modules/proc_applying-configurations.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+----
+
+Also you might want to rerun installer with some new option, so you can:
+
+[options="nowrap" subs="attributes,quotes"]
+----
+# {foreman-installer} _new options_
+----
+
+After any change, please run this quick {Project} health-check:
```
[the remark this answers is not in the document]
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: Changed, thank you

File: guides/common/modules/proc_applying-configurations.adoc (hunk 3 above)
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: Removed, thank you

File: guides/common/modules/con_performance-tuning-quickstart.adoc [hunk 5]
```diff
@@ -26,5 +26,6 @@ The default value is `default`.
 ----
 # {foreman-installer} --tuning "_My_Installation_Size_"
 ----
+. Optional: For some additional checks see xref:Applying_configurations_{context}[].
```
Comment (maximiliankolb) [inline guides/common/modules/con_performance-tuning-quickstart.adoc:29]: Please reword this. To me, it's unclear why/in which case I'd want to follow the link.

File: guides/common/modules/proc_applying-configurations.adoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
+[id="Applying_configurations_{context}"]
+= Applying configurations
+
+In following sections we suggest various tunables and how to apply them.
+Please always test changing these in non production environment first, with valid backup and with proper outage window as in most of the cases Satellite restart is required.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_applying-configurations.adoc:5]: Your PR should not contain product names like satellite. Please use attributes instead as described in CONTRIBUTING.md

File: guides/common/modules/proc_applying-configurations.adoc [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+----
+
+Note this will still need to reload the service to do the actual change. It is best to restart whole Satellite:
+
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} service restart
+----
+
+If you have changed some configuration file like `/etc/foreman-installer/custom-hiera.yaml` you need to rerun installer to apply the changes:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_applying-configurations.adoc:23]:
```suggestion
If you have changed some configuration file like `/etc/foreman-installer/custom-hiera.yaml`, you need to apply the changes:
```

File: guides/common/modules/proc_applying-configurations.adoc [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+----
+# {foreman-installer}
+----
+
+Also you might want to rerun installer with some new option, so you can:
+
+[options="nowrap" subs="attributes,quotes"]
+----
+# {foreman-installer} _new options_
+----
```
Comment (maximiliankolb) [inline guides/common/modules/proc_applying-configurations.adoc:35]:
```suggestion
```

To me, it is unclear _why_ this is part of the procedure.


Also, the whole procedure is missing `.Procedure` and a numbered list.

Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc:35]: Right, will reword it.

Regarding making it numbered list: It is not  a single procedure I would say. It is more set of steps that are connected, but based on what exactly you are changing, you need to pick right subset.

File: guides/common/modules/con_postgresql-tuning.adoc [hunk 9]
```diff
@@ -20,12 +20,7 @@ postgresql::server::config_entries:
 ----
 +
 You can use this to effectively tune down your {Project} instance irrespective of a tuning profile.
-. Rerun the `{foreman-installer}`.
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# {foreman-installer}
-----
+. Apply your changes by following xref:Applying_configurations_{context}[] section.
```
[the remark this answers is not in the document]
Comment (jhutar) [reply guides/common/modules/con_postgresql-tuning.adoc]: Thank you, fixed.

File: guides/common/modules/con_performance-tuning-quickstart.adoc (hunk 5 above)
Comment (jhutar) [reply guides/common/modules/con_performance-tuning-quickstart.adoc]: Done, thank you.

File: guides/common/modules/proc_applying-configurations.adoc (hunk 6 above)
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: Missed that. Thank you.

File: guides/common/modules/proc_applying-configurations.adoc (hunk 7 above)
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: Done. Thank you.

File: guides/common/modules/proc_applying-configurations.adoc [hunk 10]
```diff
@@ -13,21 +13,21 @@ If you have changed some systemd service file, you need to notify systemd daemon
 # systemctl daemon-reload
 ----
 
-Note this will still need to reload the service to do the actual change. It is best to restart whole Satellite:
+After this you still need to reload relevant service for the change to take effect. It is best to restart whole {Project}:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_applying-configurations.adoc:17]:
```suggestion
Restart {Project} services:
```

File: guides/common/modules/proc_applying-configurations.adoc (hunk 8 above)
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: Added, thank you!

File: guides/common/modules/proc_applying-configurations.adoc [hunk 11]
```diff
@@ -13,21 +13,21 @@ If you have changed some systemd service file, you need to notify systemd daemon
...
 
 [options="nowrap" subs="attributes"]
 ----
 # {foreman-maintain} service restart
 ----
 
-If you have changed some configuration file like `/etc/foreman-installer/custom-hiera.yaml` you need to rerun installer to apply the changes:
+If you have changed some configuration file (e.g. `/etc/foreman-installer/custom-hiera.yaml`) you need to rerun installer to apply the changes:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_applying-configurations.adoc]: We typically avoid (latin) abbreviations like that according to IBM style guide.

```suggestion
If you have changed a configuration file such as `/etc/foreman-installer/custom-hiera.yaml`, rerun installer to apply your changes:
```

File: guides/common/modules/proc_applying-configurations.adoc (hunk 10 above)
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: Thank you.

File: guides/common/modules/proc_applying-configurations.adoc [hunk 12]
```diff
@@ -0,0 +1,46 @@
+[id="Applying_configurations_{context}"]
+= Applying configurations
+
+In following sections we suggest various tunables and how to apply them.
+Please always test changing these in non production environment first, with valid backup and with proper outage window as in most of the cases {Project} restart is required.
+
+It is also a good practice to setup a monitoring before applying any change as it will allow you to evaluate effect of the change.
+Our testing environment might be too far from what you will see although we are trying hard to mimic real world environment.
+
+._Changing systemd service files_
```
[the remark this answers is not in the document]
Comment (jhutar) [reply guides/common/modules/proc_applying-configurations.adoc]: I see, thank you. Done.

**Review by maximiliankolb — APPROVED** [review]
LGTM! Let's move forward.

**Pull request merged**